wrapping/itkPyBuffer.wrap

```diff
@@ -1,21 +1,54 @@
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/PyBuffer.i.init"
                "${CMAKE_CURRENT_BINARY_DIR}/PyBuffer.i"
                @ONLY)
-
 itk_wrap_class("itk::PyBuffer")
-  UNIQUE(types "${WRAP_ITK_SCALAR};UC")
+  UNIQUE(types "${WRAP_ITK_SCALAR};UC;D;US;UI;UL;SC;SS;SI;SL;F")
   foreach(d ${ITK_WRAP_DIMS})
     foreach(t ${types})
-      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}")
-      set(PyBufferTypes ${ITKM_I${t}${d}})
       set(PixelType ${t})
-      configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/PyBuffer.i.in
+      if( DEFINED ITKT_I${t}${d} )
+        itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}")
+        set(PyBufferTypes ${ITKM_I${t}${d}})
+        configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/PyBuffer.i.in
+                        ${CMAKE_CURRENT_BINARY_DIR}/PyBuffer.i.temp
+                        @ONLY)
+        file(READ ${CMAKE_CURRENT_BINARY_DIR}/PyBuffer.i.temp
+             PyBufferInterfaceTemp)
+        file(APPEND ${CMAKE_CURRENT_BINARY_DIR}/PyBuffer.i
+             ${PyBufferInterfaceTemp})
+      endif()
+      # Wraps RGB and RGBA types that have been selected to be wrapped
+      # We have to recompute them all to extract the pixel type (component type)
```

Comment:
The reason why I did not use WRAP_ITK_COLOR and WRAP_ITK_VECTOR (comment below) is because I need to know the pixel type (more correctly the component type, e.g. F) to configure PyBuffer.i .